docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -17,5 +17,5 @@ Student -up-> "*" Tag
 Student *--> Name
 Student *--> Phone
 Student *--> Email
-Student *--> Address
+Student *--> Student
```

Comment:
This change shouldn't happen as it actually stores the Address

Reply:
Nice catch